config/gcos.words

```diff
@@ -0,0 +1,478 @@
+reserved:	COMPUTATIONAL-9=FLOAT-SHORT
+reserved:	COMPUTATIONAL-10=FLOAT-LONG
+reserved:	COMPUTATIONAL-15=FLOAT-LONG
+reserved:	COMPUTE
+reserved:	CONFIGURATION
+reserved:	CONSTANT		# COBOL 2002 != GCOS (unsupported)
+reserved:	CONTAINS
+reserved:	CONTENT
+reserved:	CONTINUE
+reserved:	CONTROL
+reserved:	CONTROLS
+reserved:	CONVERSION		# not supported yet
+reserved:	CONVERTING
+reserved:	COPY
+reserved:	CORR
+reserved:	CORRESPONDING
+reserved:	COUNT
+reserved:	CURRENCY
+reserved:	DATA
+reserved:	DATE
+reserved:	DATE-COMPILED
+reserved:	DATE-WRITTEN
+reserved:	DAY
+reserved:	DAY-OF-WEEK
+reserved:	DE
+reserved:	DEBUGGING
+reserved:	DECIMAL-POINT
+reserved:	DECLARATIVES
+reserved:	DEFAULT			# COBOL 2002 != GCOS (unsupported)
+reserved:	DELETE
+reserved:	DELIMITED
+reserved:	DELIMITER
+reserved:	DEPENDING
+reserved:	DESCENDING
+reserved:	DESTINATION
+reserved:	DETAIL
+reserved:	DISABLE
+reserved:	DISPLAY
+reserved:	DIVIDE
+reserved:	DIVISION
+reserved:	DOWN
+reserved:	DUPLICATES
+reserved:	DYNAMIC
+reserved:	EBCDIC
+reserved:	EGI
+reserved:	ELSE
+reserved:	EMI
+reserved:	ENABLE
+reserved:	END
+reserved:	END-ADD
+reserved:	END-CALL
+reserved:	END-COMPUTE
+reserved:	END-DELETE
+reserved:	END-DIVIDE
+reserved:	END-EVALUATE
+reserved:	END-IF
+reserved:	END-MULTIPLY
+reserved:	END-OF-PAGE
+reserved:	END-PERFORM
+reserved:	END-READ
+reserved:	END-RECEIVE
+reserved:	END-RETURN
+reserved:	END-REWRITE
+reserved:	END-SEARCH
+reserved:	END-START
+reserved:	END-STRING
+reserved:	END-SUBTRACT
+reserved:	END-UNSTRING
+reserved:	END-WRITE
+reserved:	ENTER
+reserved:	ENVIRONMENT
+reserved:	EOP
+reserved:	EQUAL
+reserved:	EQUALS=EQUAL
+reserved:	ERROR
+reserved:	ESI
+reserved:	EVALUATE
+reserved:	EVERY
+reserved:	EXCEEDS=GREATER
+reserved:	EXCEPTION
+reserved:	EXIT
+reserved:	EXTEND
+reserved:	EXTERNAL
+reserved:	FALSE
+reserved:	FD
+reserved:	FILE
+reserved:	FILE-CONTROL
+reserved:	FILLER
+reserved:	FINAL
+reserved:	FIRST
+reserved:	FLOAT-LONG		# COBOL 2002
+reserved:	FLOAT-SHORT		# COBOL 2002
+reserved:	FLR
+reserved:	FOOTING
+reserved:	FOR
+reserved:	FROM
+reserved:	FUNCTION
+reserved:	GENERATE
+reserved:	GIVING
+reserved:	GLOBAL
+reserved:	GO
+reserved:	GOBACK			# COBOL 2002 ~ GCOS EXIT PROGRAM GIVING
+reserved:	GREATER
+reserved:	GROUP
+reserved:	HEADING
+reserved:	HIGH-VALUE
+reserved:	HIGH-VALUES=HIGH-VALUE
+reserved:	I-O
+reserved:	I-O-CONTROL
+reserved:	IDENTIFICATION
+reserved:	IF
+reserved:	IN
+reserved:	INDEX
+reserved:	INDEXED
+reserved:	INDICATE
+reserved:	INITIAL
+reserved:	INITIALIZE
+reserved:	INITIATE
+reserved:	INPUT
+reserved:	INPUT-OUTPUT
+reserved:	INSPECT
+reserved:	INSTALLATION
+reserved:	INTO
+reserved:	INVALID
+reserved:	IS
+reserved:	JUST
+reserved:	JUSTIFIED
+reserved:	KEY
+reserved:	LABEL
+reserved:	LAST
+reserved:	LEADING
+reserved:	LEFT
+reserved:	LENGTH
+reserved:	LESS
+reserved:	LIMIT
+reserved:	LIMITS
+reserved:	LINAGE
+reserved:	LINAGE-COUNTER
+reserved:	LINE
+reserved:	LINE-COUNTER
+reserved:	LINE-SEQUENTIAL*	# GNUCobol
```

Comment:
If this isn't supported in GCOS just drop it (the lax config will additionally use the default words).

Reply:
Ok I dropped this.